src/wx/faudio.cpp

```diff
@@ -339,15 +335,15 @@ bool FAudio_Output::init(long sampleRate)
     if (failed || initialized)
         return false;
```

Comment:
Would it make sense to assert here instead?

Reply:
That I wouldn't know, that's something that is even in the original xaudio source file which this is largely based off of.

Reply:
Ack.